spec/functions/length_spec.rb

```diff
@@ -26,10 +26,6 @@
   it { is_expected.to run.with_params('āβćđ').and_return(4) }
 
   context 'when using a class extending String' do
-    it 'should call its size method' do
-      value = AlsoString.new('asdfghjkl')
-      value.expects(:length).returns('foo')
```

Comment:
I'm not sure it ever made any sense to mock returning a string when the function should only ever return an int.